llvm/lib/Target/DirectX/DXILLegalizePass.cpp

```diff
@@ -405,6 +405,20 @@ static void removeMemSet(Instruction &I,
   ToRemove.push_back(CI);
 }
 
+static void updateFnegToFsub(Instruction &I,
+                             SmallVectorImpl<Instruction *> &ToRemove,
+                             DenseMap<Value *, Value *> &ReplacedValues) {
```

Comment:
since we don't use `ReplacedValues` in this function  we typically don't define the argument.
```suggestion
                             DenseMap<Value *, Value *> &) {
```